lib/models/concerts.js

```diff
@@ -0,0 +1,40 @@
+const { ObjectID } = require("mongodb");
+const { getCollection } = require("../database");
+
+const collectionName = "concerts";
+
+async function postConcert(concert) {
+  const concertsCollection = await getCollection(collectionName);
+  const result = await concertsCollection.insertOne({ ...concert });
+
+  return result.insertedId;
+}
+
+async function getConcert(concertId) {
+  const concertsCollection = await getCollection(collectionName);
+  const objectId = new ObjectID.createFromHexString(concertId);
+  const concert = await concertsCollection.findOne({ _id: objectId });
+
+  if (!concert) {
+    throw new Error("Cannot find concerts");
+  }
+
+  return concert;
+}
+
+// .find().sort([['_id', 1]]).limit(5).select('title  _id').exec(function(e, data)
+
+async function getConcerts() {
+  const concertsCollection = await getCollection(collectionName);
+  const concerts = await concertsCollection.find().sort({ _id: -1 }).toArray();
```

Comment:
`.sort({ _id: -1 })` doesn't make sense because the `_id` is autogenerated and hashed. It is totally random now :).

Reply:
I want to display the latest concerts first and with this, it works 🤔 
@lmachens